soft_introduction.html

```diff
@@ -125,6 +125,30 @@ <h1>Projekty</h1>
                 <section>
                     <img src="img/alinka.png" style="background:none; border:none; box-shadow:none; "/>
                     <a href="https://alinka.io/">https://alinka.io</a>
+                    <aside class="notes">
+                        <p>Desktopowa aplikacja, która wspiera wydawanie orzeczeń, wraz z innymi niezbędnymi dokumentami przez polskie Poradnie Psychologiczno - Pedagogiczne.
+                        Alinka jest aplikacja standalone zbudowaną na silniku Chrome.</p>
+                        <p>Aktualne funkcjonalności:</p>
+                        <ul>
+                            <li>Pasek progresu wystawiania orzeczenia</li>
+                            <li>Tworzenie bazy danych - imię i nazwisko oraz adres zamieszkania dziecka, rodzica/rodziców/opiekunów prawnych</li>
+                            <li>Możliwość wyboru rodzaju orzeczenia: o potrzebie kształcenia specjalnego, o potrzebie indywidualnego rocznego, o potrzebie indywidualnego nauczania, o potrzebie wczesnego wspomagania rozwoju</li>
+                            <li>Możliwość wyboru przyczyny wydawania orzeczenia: np. niedowidzenie, autyzm</li>
+                            <li>Możliwość wyboru placówki – np. szkoła podstawowa, szkoła średnia – dane zaciągane z GUS</li>
+                            <li>Generowania dokumentu orzeczenia</li>
+                        </ul>
+                        <p>Aktualne prace skupiają się na dostarczeniu gotowej aplikacji, która wesprze pracę Poradni
+                        z Wrześni i Grodziska Wlkp. w generowaniu dokumentów:</p>
+                        <ul>
+                            <li>Metryka sprawy</li>
+                            <li>Protokół</li>
+                            <li>Zarządzenie</li>
+                            <li>Zawiadomienie o ZO</li>
```

Comment:
podałabym pełną nazwę tego skrótu - nie mam pojęcia od czego to ZO :)

Reply:
ZO- Zespół orzekający